test/client/fixtures/sandbox/node/attributes-test.js

```diff
@@ -483,6 +483,36 @@ test('element.innerHTML', function () {
     checkElement($container.find('script')[0], 'src', '!s');
 });
 
+test('SVGImageElement with an existing xlink:href should contain only one stored href attribute after href.baseVal is changed', function () {
+    var div               = document.createElement('div');
+    var xlinkNamespaceUrl = 'http://www.w3.org/1999/xlink';
+    var baseVal           = 'http://example.com/test.svg';
+
+    // NOTE: IE11 adds extra 'ns1:' namespace to stored xlink href attribute during processing,
+    // so we use div.innerHTML = html with existed xlink:href-hammerhead-stored-value attribute for all cases.
+    var html = '<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="' + xlinkNamespaceUrl + '">\n' +
+               '<image xlink:href="' + baseVal + '" xlink:href-hammerhead-stored-value="' + baseVal + '"></image>\n' +
+               '</svg>';
+
+    div.innerHTML = html;
```

Comment:
After setting the `innerHTML` property we have a wrong markup for `xlink:href-hammerhead-stored-value` attribute:

![image](https://user-images.githubusercontent.com/4133518/38411785-8932fff4-3990-11e8-98fb-08f7008f16ea.png)

It's a bug. Need to fix it on hammerhead side.